curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-question-e.md

```diff
@@ -12,7 +12,7 @@ The `flex` declaration is actually a shorthand for 3 properties that you can set
 
 In this case, `flex` is actually a shorthand for `flex-grow`, `flex-shrink` and `flex-basis`.
 
-<img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/0cc6b26bb0c4b94524369d327c97a8fb11e83b6b/foundations/html_css/flexbox/imgs/10.png" alt="flex shorthand">
+<img src="https://cdn.freecodecamp.org/curriculum/odin-project/flex-box/flexbox-04.png" alt="The image shows a snippet of CSS code on a dark background. It defines a style for div elements, setting their flex property to 1. This is a common CSS Flexbox property that allows a div to grow or shrink to fill the available space in a flex container.">
```

Comment:
No need to over describe this image, especially since the content below describes it as well. Also, alt text should not contain additional information that isn't provided to everyone else. If screen reader users need to know that the flexbox property allows a div to grow or shrink to fill the available space in a flex container then everyone needs to know that and it should be in the actual content. Alt text should only describe whats in the image.

I would suggest the following brief alt text:

"CSS code setting the flex property to 1 for a div element."